.github/workflows/rector_ci.yaml

```diff
@@ -45,43 +41,18 @@ jobs:
 
             -   run: composer install --no-progress --ansi
 
-            ## First run Rector - here can't be --dry-run !!! it would stop the job with it and not commit anyhting in the future
-            -   run: bin/rector process ${{ matrix.directories }} --config rector-ci.php --ansi --no-progress-bar
+            ## First run Rector - here can't be --dry-run !!! it would stop the job with it and not commit anything in the future
+            -   run: bin/rector rectify ${{ matrix.directories }} --config rector-ci.php --ansi --no-progress-bar
+            -   run: vendor/bin/ecs check ${{ matrix.directories }} --fix --ansi
 
             -
-                name: Check for Rector modified files
-                id: rector-git-check
-                run: echo ::set-output name=modified::$(if git diff --exit-code --no-patch; then echo "false"; else echo "true"; fi)
-
-            -   name: Git config
-                if: steps.rector-git-check.outputs.modified == 'true'
-                run: |
-                    git config --global user.name 'rector-bot'
-                    git config --global user.email 'tomas@getrector.org'
-                    echo ::set-env name=COMMIT_MESSAGE::$(git log -1 --pretty=format:"%s")
-
-            -   name: Commit Rector changes
-                if: steps.rector-git-check.outputs.modified == 'true'
-                run: git commit -am "[rector] ${COMMIT_MESSAGE}"
-
-            ## Now, there might be coding standard issues after running Rector
-            -
-                if: steps.rector-git-check.outputs.modified == 'true'
-                run: composer fix-cs
-
-            -
-                name: Check for CS modified files
-                if: steps.rector-git-check.outputs.modified == 'true'
-                id: cs-git-check
-                run: echo ::set-output name=modified::$(if git diff --exit-code --no-patch; then echo "false"; else echo "true"; fi)
-
-            -   name: Commit CS changes
-                if: steps.cs-git-check.outputs.modified == 'true'
-                run: git commit -am "[cs] ${COMMIT_MESSAGE}"
-
-
-            -   name: Push changes
-                if: steps.rector-git-check.outputs.modified == 'true'
-                run: git push
-                # In case we want to fail this job when there are changed files, just add "exit 1"
-
```

Comment:
@JanMikes I've discovered very cool Github Action, that handles all the commiting for us!
https://github.com/EndBug/add-and-commit

We're running today for:

- doctrine/annotation sync
- documntation generator
- monorepo sync
- Rector

And so far so good :)

Reply:
Thx for showing me, it shortens the syntax a little 😄 let's see how it behaves in some edge cases that might have been covered by us but not in action and maybe scenarios like that are completely okay 👍

Reply:
Indeed, it's testing time this week :)

I look for some other action, that will stop if there is no change in previous action. So the coding standard does not run on unchanged files

Reply:
**Update**: this is not Github Action, but solves it :) 
 https://github.com/rectorphp/rector/pull/4487/files#diff-3f20cc93285dee3e6dc5886aa6f9c5f8fb9713b57d575f099947fc8d86153c0c